service/src/main/java/bio/terra/cbas/runsets/monitoring/RunSetAbortManager.java

```diff
@@ -29,18 +29,15 @@ public RunSetAbortManager(RunSetDao runSetDao, RunDao runDao, CromwellService cr
     this.cromwellService = cromwellService;
   }
 
-  public AbortRequestDetails abortRunSet(UUID runSetId, BearerToken userToken) {
+  public AbortRequestDetails abortRunSet(RunSet runSet, BearerToken userToken) {
```

Comment:
The reason this was updated was because now in the `abortRunset` handler in `RunSetsApiController` it [fetches the run set details from the database](https://github.com/DataBiosphere/cbas/pull/281/files#diff-1bd3bc7318e38eee77c2fce2eff273155ad4195dcb9bed577968795ee1baac6fR298-R299) to check if the run set is in Queued state. As a result, I updated this method since we already have the RunSet object and it doesn't need be fetched again on [L37](https://github.com/DataBiosphere/cbas/pull/281/files#diff-8f3866ed117fc606d74801b93514dd66795f764a6570ed289f7c768e3a66c1ffL37-L39).